src/coreclr/src/jit/codegenarm64.cpp

```diff
@@ -2666,7 +2666,7 @@ void CodeGen::genCodeForCpObj(GenTreeObj* cpObjNode)
     if (cpObjNode->gtFlags & GTF_BLK_VOLATILE)
     {
         // issue a INS_BARRIER_ISHLD after a volatile CpObj operation
```

Comment:
I think this comment should be changed to `// This requires only a load barrier.`